tests/test_mock_controller.py

```diff
@@ -67,15 +72,26 @@ async def asyncSetUp(self) -> None:
 
         self.log = logging.getLogger("MockTestCase")
 
-    async def read(self) -> dict:
-        """Utility function to read a string from the reader and unmarshal it
+    async def read(self, timeout: float = DEFAULT_TIMEOUT) -> dict:
+        """Utility function to read a string from the reader and unmarshal it.
+
+        Parameters
+        ----------
+        timeout : float, optional
+            The timeout to use; default to 1 [s].
 
         Returns
         -------
         data : `dict`
             A dictionary with objects representing the string read.
+
+        Notes
+        -----
+        The timeout should only be set in case a network issue is mocked.
```

Comment:
I'm somewhat confused by this statement. What do you mean by a "network issue"?

I see that you use a longer value when simulating a slow network where you are willing to wait for the data. If that's what you have in mind, you could say that here, but:
* You have nice explanatory code comments where you do that below.
* It is a fairly self-evident use case.

So I suggest deleting the note. But if you are keen to keep it, then please clarify it.

Reply:
I removed the note.